nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -12,9 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <string>
+#include <memory>
+#include <set>
 #include <sstream>
```

Comment:
Looks like we don't need this anymore.

Reply:
@orduno @crdelsey  Not part of this PR, but I'm investigating utilities to detect unneeded includes.